src/com/epam/task/module4/theSimplestClassesAndObjects/task8/Customers.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.task.module4.theSimplestClassesAndObjects.task8;
+
+import com.epam.task.Utils.InputUtil;
+import com.epam.task.module4.theSimplestClassesAndObjects.task8.Comparator.ComparatorByLastName;
+
+import java.util.Arrays;
+
+public class Customers {
+    public static Customer[] getCustomers() {
+        Customer[] customers = new Customer[5];
```

Comment:
It should be aggregated in the class, not in the method. Please, move the customers array to a class field. Use 'getCustomers' to have access to the customers field and pass the array itself in the constructor of the 'Customers'.